src/lib/core/setup.c

```diff
@@ -1888,8 +1888,9 @@ bt_setup_get_unique_machine_id (const BtSetup * const self,
 
   gchar *const id = g_strdup_printf ("%s 00", base_name);
   gchar *const ptr = &id[strlen (base_name) + 1];
+  int ptr_len = strlen(ptr);
   do {
-    (void) g_sprintf (ptr, "%02u", i++);
+    g_snprintf (ptr, ptr_len, "%02u", i++);
```

Comment:
please keep the void to tell tools that we knowingly ignore the return value

Reply:
Thanks for explaining that -- I'll revert.